test_package/conanfile.py

```diff
@@ -1,18 +1,14 @@
-#!/usr/bin/env python
```

Comment:
templates...

Reply:
conanfile's are not scripts, so the templates should be modified.
Agreed?

Reply:
https://github.com/bincrafters/conan-templates/pull/49

Reply:
Merged into the templates